src/ZkFold/Symbolic/Cardano/Contracts/BatchTransfer.hs

```diff
@@ -29,23 +28,11 @@ type Tx context = Transaction 6 0 11 Tokens 0 () context
 hash :: forall context x . (Symbolic context, MiMCHash (BaseField context) context x) => x -> FieldElement context
 hash = mimcHash @(BaseField context) mimcConstants zero
 
-type Sig context =
+verifySignature ::
+    forall context .
     ( Symbolic context
-    , StrictConv (context Par1) (UInt 256 Auto context)
-    , FromConstant Natural (UInt 256 Auto context)
-    , MultiplicativeSemigroup (UInt 256 Auto context)
     , MiMCHash (BaseField context) context (TxOut context, TxOut context)
```

Comment:
Can this be inferred from `Symbolic context` and, maybe, `FromConstant (BaseField context) (FieldElement context)`?